koku/masu/util/azure/common.py

```diff
@@ -35,8 +35,6 @@
     "PreTaxCost",
     "ConsumedService",
     "ResourceType",
-    "InstanceId",
-    "OfferId",
     "AdditionalInfo",
```

Comment:
so why are we removing these columns again?

Reply:
I think I'm just struggling with why some of these changes are necessary, Azure Docs don't indicate a column called `SubscriptionGuid` exists, just `SubscriptionId`: https://learn.microsoft.com/en-us/azure/cost-management-billing/automate/understand-usage-details-fields

I'm also not sure we can rely on specific casing like `MeterSubCategory` vs `MeterSubcategory` since the docs above state casing can be inconsistent:
```
Some fields might differ in casing and spacing between account types. Older versions of pay-as-you-go cost details files have separate sections for the statement and daily cost.
```

I know Azure has different report formats such as MCA vs EA vs PayG but where are we getting the information for these different columns from?

Reply:
Related to the casing differences, in the post processors we try to ignore casing to be as consistent as possible like here: https://github.com/project-koku/koku/blob/main/koku/masu/util/azure/azure_post_processor.py#L61-L85

Would it be possible to handle all kinds of casing like we do in the post processor rather than create more case variants to match?

Reply:
I think you were actually handling the casing differences like I was hoping and I was just overlooking it. I think removing the lower is less consistent here: https://github.com/project-koku/koku/pull/4616/commits/6c4757f0d65f3f34062e9971dcbdff75ebaabfb3

That one is totally my bad, I apparently need to get my eyes checked